gulpfile.js/import/importRoadmap.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.ä
+ */
+
+'use strict';
+
+require('module-alias/register');
+
+const fs = require('fs');
+const yaml = require('js-yaml');
+const emojiStrip = require('emoji-strip');
+const {promisify} = require('util');
+const writeFileAsync = promisify(fs.writeFile);
+const {GitHubImporter} = require('@lib/pipeline/gitHubImporter');
+const log = require('@lib/utils/log')('Import Working Groups');
+
+/* The GitHub organisation the repositories imported from are located */
+const WG_GH_ORGANISATION = 'ampproject';
+/* Path where the roadmap data gets imported to */
+const ROADMAP_DIRECTORY_PATH = 'pages/shared/data';
```

Comment:
Use the `project.absolute` utility here to make sure it's the correct path no matter from where you are calling the task.